packages/stripe_platform_interface/lib/src/models/platform_pay.dart

```diff
@@ -400,3 +383,30 @@ class ApplePayMultiMerchant with _$ApplePayMultiMerchant {
   factory ApplePayMultiMerchant.fromJson(Map<String, dynamic> json) =>
       _$ApplePayMultiMerchantFromJson(json);
 }
+
+@freezed
+
+/// Parameters related to order details with Apple pay
+///
+/// At this moment only Apple pay is supported.
+/// Similar to [KPaymentOrderDetails]
+/// See https://stripe.com/docs/apple-pay?platform=ios&locale=es-ES#order-tracking
+class PlatformPayOrderDetails with _$PlatformPayOrderDetails {
+  @JsonSerializable(explicitToJson: true)
+  const factory PlatformPayOrderDetails.applePay({
+    /// eg: "com.myapp.order"
```

Comment:
Nit:  I think we should also add some explanatory description in the docs to it. I will do this in the sync branch